chitter.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+
+class Chitter 
+
+  attr_reader :message, :id, :time
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      con = PG.connect :dbname => 'chitter_test'
+    else 
+      con = PG.connect :dbname => 'chitter'
+    end
+    retrieve_all = con.exec "SELECT * FROM peeps"
+    retrieve_all.map do |column|
+      Chitter.new(message: column['message'], id: column['id'], time: column['time'])
+    end
+
+  end
+
+  def self.add(message)
```

Comment:
Remember that SQL doesn't particularly like apostrophes? I ran into this same bug in my own code. I'd LOVE to be able to use an apostrophe in my peep!